apps/masterbots.ai/components/routes/browse/browse-chat-message-list.tsx

```
@@ -32,11 +47,12 @@ export function BrowseChatMessageList({
   return (
     <div>
       {pairs.map((pair: MessagePair, key: number) => (
-        <ChatAccordion
+        <BrowseAccordion
           defaultState
           key={key}
           isOpen={key == 0}
```

Biome

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)